src/Artsy/Router/Utils/Route.tsx

```diff
@@ -28,33 +27,8 @@ function createRender({
 }: CreateRenderProps) {
   return (renderArgs: RenderArgProps) => {
     const { Component, props, error } = renderArgs
-    let status: number
-    let message: string
-    if (error) {
-      if (error instanceof HttpError) {
-        throw error
-      } else if (error.name === "RRNLRequestError") {
-        // TODO: Better error typing.
-        // @ts-ignore
-        const firstError = get(error, e => e.res.errors[0])
-        const statusCodes = get(
-          firstError,
-          e => e.extensions.httpStatusCodes,
-          []
-        )
-        if (statusCodes.length === 1) {
-          status = statusCodes[0]
-          message = firstError.message
-        }
-      } else {
-        status = 500
-        message = error.message
-      }
-
-      // TODO: Need upstream fix type in found as it complains about missing
-      // second argument.
-      // @ts-ignore
-      throw new HttpError(status, message)
+    if (error && error instanceof HttpError) {
```

Comment:
We throw an error of this type [here](https://github.com/artsy/reaction/blob/424e58addba47af4ed30ecedc753d32f100d4049/src/Artsy/Relay/createRelaySSREnvironment.ts#L115), only when there is an underlying principal field error w/ a valid HTTP status code.

Otherwise, no more throwing!